README.md

```diff
@@ -22,12 +22,11 @@ pip install -e .
 ```
 
 ## Data download
-Light curve feature data for ZTF DR3 fields used in the research are available on SNAD SAI MSU server:
+Light curve feature data for ZTF DR3 fields used in the research are available on [Zenodo](https://zenodo.org/record/4318700).
+You can download it by:
```

Comment:
Please note, that the script is a convenient way to download the data, but not the single one possible way to do so.